From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Qodo AI Assistant <qodo@example.com>
Date: Mon, 18 Nov 2024 19:25:00 +0000
Subject: [PATCH] blackpill: remove broken pinctrl include

The stm32f411c(c-e)ux-pinctrl.dtsi file doesn't exist and the glob
pattern doesn't work with the C preprocessor.

Upstream-Status: Inappropriate [board definition issue]

Signed-off-by: Qodo AI Assistant <qodo@example.com>
---
 zephyr/boards/weact/blackpill_f411ce/blackpill_f411ce.dts | 1 -
 1 file changed, 1 deletion(-)

--- a/zephyr/boards/weact/blackpill_f411ce/blackpill_f411ce.dts
+++ b/zephyr/boards/weact/blackpill_f411ce/blackpill_f411ce.dts
@@ -7,7 +7,6 @@
 
 /dts-v1/;
 #include <st/f4/stm32f411Xe.dtsi>
-#include <st/f4/stm32f411c(c-e)ux-pinctrl.dtsi>
 #include <zephyr/dt-bindings/input/input-event-codes.h>
 
 / {
